jina/orchestrate/deployments/__init__.py

```diff
@@ -101,6 +101,13 @@ def head_port(self):
         """
         return self.head_args.port if self.head_args else None
 
+    @property
+    def head_port_monitoring(self):
```

Comment:
why is this property relevant?

Reply:
Well there is one for the port so it could be usefull to have one fort port_monitoring. It think that it is mainly used in test for now